app/models/parent_object.rb

```diff
@@ -215,9 +215,20 @@ def ready_for_manifest?
     !child_objects.pluck(:width).include?(nil)
   end
 
-  def representative_thumbnail
-    oid = child_objects.where(order: 1)&.first&.oid
-    "#{ENV['IIIF_IMAGE_BASE_URL']}/2/#{oid}/full/!200,200/0/default.jpg"
+  def representative_child
+    if representative_child_oid
+      begin
```

Comment:
use find_by_id instead here and skip the rescue. that way you only need the else block

```
@representative_child = if representative_child_oid
  child_objects.find_by(id: representative_child_oid)
end
@representative_child ||= child_objects.where(order: 1)&.first

Reply:
Cool, Mark and I were thinking there must be a way but hadn't hit on it, thanks for that.